### PR TITLE
Mixpanel

### PR DESCRIPTION
**Pull request by dimarconicola**
**Description**
We want to track users' behavior on the website. In order to do so, I've created Mixpanel tracking events for the main interactions

**Checklist**
 [x] Changes don't break existing behavior
 [x] Tests coverage hasn't decreased

**Testing**
I've checked that the events appear on the "live" section on Mixpanel

### CODE REVIEW

**Comment by Alexintosh** [comment]
Use proper format for PR @dimarconicola

**Review by Alexintosh — APPROVED** [review]

File: src/stores/eth.js [hunk 1]
```diff
@@ -11,14 +11,14 @@ const notify = Notify({
 
 const eth = store({
   account: undefined,
-  awp: '0x70d1c8c8300a785ee4efe74db4d2972a7e669cf6',
-  awpX: '0xF640d521793B5BF474fD4051543E9D238f108334',
+  awp: '0x7cb095e839c2da9fc693435ac4e9dc0b25bd8e74',
+  awpX: '0xb28d6F7A62Dd1B7b39fAC5e67DBf1dBe3653AEF6',
```
Comment (MickdeGraaf) [inline src/stores/eth.js:15]: Why is this in this PR?

File: src/stores/eth.js [hunk 2]
```diff
@@ -11,14 +11,14 @@ const notify = Notify({
...
   setIssuanceModule: '0x91E1489D04054Ae552a369504F94E0236909c53c',
   setTransferProxy: '0x61d264865756751392C0f00357Cc26ea70D98E3B',
   bntIndex: '0xf09a28dcf6e78c1d41c007e9c455690b9caa856a',
   btcIndex: '0x8ff461ce88b24eed98626d31ebc4dad4c3766909',
-  dai: '0x1d7e3a1a65a367db1d1d3f51a54ac01a2c4c92ff',
-  daiX: '0x1a3e8f0A53E0524fC2DF6841408abe778C8Ecd9d',
+  dai: '0xff2c8f09c8b20847972db158d8c63e45f026796f',
+  daiX: '0xc3a4e9a3adc7e128a43a5da6dac5677b90a17d01',
```
Comment (MickdeGraaf) [inline src/stores/eth.js:21]: See above

File: src/stores/exchangeModal.js [hunk 3]
```diff
@@ -205,6 +205,10 @@ const exchangeModal = store({
 
     const value = BigNumber(evt.target.value);
 
+    if (value.isZero()) {
+      return;
+    }
+
```
Comment (MickdeGraaf) [inline src/stores/exchangeModal.js:211]: Not related to mixpanel

File: src/stores/exchangeModal.js [hunk 4]
```diff
@@ -263,6 +267,10 @@ const exchangeModal = store({
...
```
Comment (MickdeGraaf) [inline src/stores/exchangeModal.js:273]: See above

File: src/stores/myAccount.js [hunk 5]
```diff
@@ -8,6 +8,10 @@ import eth from './eth';
 import erc20 from '../abi/erc20';
 import gun from './gun';
 
+const airdropEndpoint = (address) => (
+  `https://pie-protocol-api.herokuapp.com/portfolio/airdrop/${address}`
+);
+
```
Comment (MickdeGraaf) [inline src/stores/myAccount.js:14]: See above

**Comment by MickdeGraaf** [comment]
There is some stuff in this PR that is not related to mixpanel and that might have some unwanted consequences. Otherwise it looks good to me

**Comment by dimarconicola** [comment]
This PR was rebased twice, I think that's why there's code that's not related to Mixpanel

**Pull request closed without merging**